### PR TITLE
Class: fix ble bug.

### PR DESCRIPTION
**Pull request by lior7**

### CODE REVIEW

File: Class.h [hunk 1]
```diff
@@ -3,6 +3,6 @@
 
 @interface Class : NSObject
 
-@property (nonatomic) BOOL blahBlah;
+@property (nonatomic) BOOL blahBle;
```
Comment (xorserer) [inline Class.h]: Add documentation.

File: Class.h [hunk 2]
```diff
@@ -3,6 +3,6 @@
 
 @interface Class : NSObject
```
Comment (xorserer) [inline Class.h]: Also add docs here.

File: Class.h (hunk 1 above)
Comment (xorserer) [inline Class.h]: Name should be `bleble`.

**Comment by xorserer** [comment]
Round done.

File: Class.h (hunk 1 above)
Comment (lior7) [inline Class.h]: done and done

File: Class.h (hunk 2 above)
Comment (lior7) [inline Class.h]: done

**Comment by lior7** [comment]
round done

**Comment by xorserer** [comment]
Squash & rebase.

**Comment by lior7** [comment]
done

**Pull request merged**